### PR TITLE
30 version bump

### PR DESCRIPTION
**Pull request by IanGrimstead**
* Version bump to `lxml` library c/o dependabot
* Update to R scripts, to reduce memory usage
* Correction to `runner-startup.sh` to ensure QA files are sent to GCP bucket

### CODE REVIEW

**Comment by IanGrimstead** [comment]
This will become release `v1.1.1` (cute!) as its a minor fix rather than new functionality

File: cloud/vm/README_RScripts.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+1. [data_impute_run.R](#data_impute_runr)
+2. [data_seats_run.R](#data_seats_runr)
+3. [data_impute_and_seats_functions.R](#data_impute_and_seats_functionsr)
+
+## `data_impute_run.R`
+
+This script is in charge of downloading data from BigQuery, tidying and imputing the data and outputting QA files.
+Each location is processed separately and will be stored in separate `.Rda` files, e.g. `TfL-images.Rda`. The CSV
+file `list_of_location_datasets.csv` will list the location names of datasets currently being processed weekly and
+their corresponding latest date in each dataset. This file isused so the R script knows which locations to download
```
Comment (Ryan2Y79) [inline cloud/vm/README_RScripts.md:15]: 'is used' not 'isused'

**Review by Ryan2Y79 — CHANGES_REQUESTED** [review]
Hey, just spotted a single spelling mistake in the README. Just amend and all is good.

File: cloud/vm/README_RScripts.md (hunk 1 above)
Comment (IanGrimstead) [reply cloud/vm/README_RScripts.md:15]: Good spot!

**Pull request merged**